### PR TITLE
conditional rendering page translation

### PR DESCRIPTION
**Pull request by sanchit36**
Translating the conditional rendering page.
Related to #168 
Review Progress

- [ ] Conditionally including JSX
- [ ]  Conditional (ternary) operator (`? :`)
- [ ] Logical AND operator (`&&`)
- [ ] Conditionally assigning JSX to a variable 
- [ ] Show an icon for incomplete items with `? :`
- [ ]  Show the item importance with `&&`
- [ ] Refactor a series of `? :` to `if` and variables

I have done till `Conditionally assigning JSX to a variable`.

### CODE REVIEW

File: beta/src/pages/learn/conditional-rendering.md [hunk 1]
```diff
@@ -309,46 +309,45 @@ export default function PackingList() {
...
-For example, a common mistake is to write code like `messageCount && <p>New messages</p>`. It's easy to assume that it renders nothing when `messageCount` is `0`, but it really renders the `0` itself!
+उदाहरण के लिए, एक सामान्य गलती `messageCount && <p>New messages</p>` जैसा कोड लिखना है। यह मान लेना आसान है कि `messageCount` `0` होने पर यह कुछ भी रेंडर नहीं करता है, लेकिन यह वास्तव में `0` को ही रेंडर करता है!
 
-To fix it, make the left side a boolean: `messageCount > 0 && <p>New messages</p>`.
+इसे ठीक करने के लिए, बाईं ओर एक बूलियन बनाएं: `messageCount > 0 && <p>New messages</p>`।
 
 </Gotcha>
 
-### Conditionally assigning JSX to a variable {/*conditionally-assigning-jsx-to-a-variable*/}
+### कंडीशनली एक वैरिएबल को JSX असाइन कर रहा है {/*conditionally-assigning-jsx-to-a-variable*/}
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:326]: `कर रहा है` => `करना`

File: beta/src/pages/learn/conditional-rendering.md [hunk 2]
```diff
@@ -309,46 +309,45 @@ export default function PackingList() {
...
 
-When the shortcuts get in the way of writing plain code, try using an `if` statement and a variable. You can reassign variables defined with [`let`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let), so start by providing the default content you want to display, the name:
+जब शॉर्टकट सादा कोड लिखने के रास्ते में आ जाते हैं, तो एक `if` स्टेटमेंट और एक वेरिएबल का उपयोग करने का प्रयास करें। आप [`let`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let) के साथ डिफाइंड वेरिएबल्स को फिर से असाइन कर सकते हैं, इसलिए डिफ़ॉल्ट कंटेंट प्रदान करके शुरु करें जिसे आप डिस्प्ले करना चाहते हैं , नाम:
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:328]: `सादा` => `प्लेन`

Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:328]: `करने का प्रयास करें` => `करके देखें`

Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:328]: `प्रदान` => `देके`

let me know your thoughts on this

File: beta/src/pages/learn/conditional-rendering.md [hunk 3]
```diff
@@ -309,46 +309,45 @@ export default function PackingList() {
...
+यदि `isPacked` `true` है, तो JSX एक्सप्रेशन को `itemContent` पर फिर से असाइन करने के लिए `if` स्टेटमेंट का उपयोग करें:
 
 ```js
 if (isPacked) {
   itemContent = name + " ✔";
 }
 ```
 
-[Curly braces open the "window into JavaScript".](/learn/javascript-in-jsx-with-curly-braces#using-curly-braces-a-window-into-the-javascript-world) Embed the variable with curly braces in the returned JSX tree, nesting the previously calculated expression inside of JSX:
+[कर्ली ब्रेसिज़ खोलते हैं "विंडो इन JavaScript"।](/learn/javascript-in-jsx-with-curly-braces#using-curly-braces-a-window-into-the-javascript-world)लौटाए गए JSX ट्री में कर्ली ब्रेसिज़ वाले वेरिएबल को एम्बेड करें, JSX के अंदर पहले से कॅल्क्युलेटेड एक्सप्रेशन को नेस्ट करें:
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:342]: `कर्ली ब्रेसिज़ खोलते हैं "विंडो इन JavaScript"।` => `कर्ली ब्रेसिज़ "जावास्क्रिप्ट के लिए एक रास्ता" खोलते है।`

Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:342]: `)लौटाए गए JSX` => `) लौटाए गए JSX`

add space after the closing bracket

Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:342]: `कॅल्क्युलेटेड` => `कैलकुलेट की हुई`

File: beta/src/pages/learn/conditional-rendering.md [hunk 4]
```diff
@@ -309,46 +309,45 @@ export default function PackingList() {
...
 
 ```js
 <li className="item">
   {itemContent}
 </li>
 ```
 
-This style is the most verbose, but it's also the most flexible. Here it is in action:
+यह शैली सबसे अधिक क्रियात्मक है, लेकिन यह सबसे लचीली भी है। यहाँ यह क्रिया में है:
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:350]: `शैली` => `तरीका`

Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:350]: `क्रियात्मक` => `वर्बोस`

lets keep it like this for now.
I can't think of any simpler word for this

Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:350]: `लचीली` => `फ्लैक्सीबल`

Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:350]: `क्रिया` => `इस्तेमाल`

File: beta/src/pages/learn/conditional-rendering.md [hunk 5]
```diff
@@ -436,16 +435,16 @@ export default function PackingList() {
 
 </Sandpack>
 
-If you're not familiar with JavaScript, this variety of styles might seem overwhelming at first. However, learning them will help you read and write any JavaScript code -- and not just React components! Pick the one you prefer for a start, and then consult this reference again if you forget how the other ones work.
+यदि आप JavaScript से परिचित नहीं हैं, तो इस तरह की शैलियाँ पहली बार में भारी लग सकती हैं। हालांकि, उन्हें सीखने से आपको किसी भी JavaScript कोड को पढ़ने और लिखने में मदद मिलेगी - न कि केवल React कौम्पोनॅन्ट! शुरुआत के लिए आप जो पसंद करते हैं उसे चुनें, और फिर इस संदर्भ को फिर से देखें यदि आप भूल जाते हैं कि दूसरे कैसे काम करते हैं।
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:438]: `शैलियाँ` => `तरीके`

Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:438]: `फिर इस संदर्भ को फिर से देखें यदि आप भूल जाते हैं कि दूसरे कैसे काम करते हैं।` => `अगर आप भूल जाते हैं कि दूसरे कैसे काम करते हैं तो इस रिफरेन्स को फिर से देखें।`

File: beta/src/pages/learn/conditional-rendering.md [hunk 6]
```diff
@@ -436,16 +435,16 @@ export default function PackingList() {
...
 
 <Recap>
 
-* In React, you control branching logic with JavaScript.
-* You can return a JSX expression conditionally with an `if` statement.
-* You can conditionally save some JSX to a variable and then include it inside other JSX by using the curly braces.
-* In JSX, `{cond ? <A /> : <B />}` means *"if `cond`, render `<A />`, otherwise `<B />`"*.
-* In JSX, `{cond && <A />}` means *"if `cond`, render `<A />`, otherwise nothing"*.
-* The shortcuts are common, but you don't have to use them if you prefer plain `if`.
+* React में, आप JavaScript के साथ ब्रांचिंग लॉजिक को नियंत्रित करते हैं।
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:442]: `नियंत्रित` => `कण्ट्रोल`

File: beta/src/pages/learn/conditional-rendering.md [hunk 7]
```diff
@@ -436,16 +435,16 @@ export default function PackingList() {
...
+* आप एक `if` स्टेटमेंट के साथ कंडिशनली रूप से JSX एक्सप्रेशन रेतुर्न कर सकते हैं।
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:443]: `कंडिशनली` => `कंडिशनल`

Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:443]: `रेतुर्न` => `रिटर्न`

File: beta/src/pages/learn/conditional-rendering.md [hunk 8]
```diff
@@ -436,16 +435,16 @@ export default function PackingList() {
...
+* आप कंडिशनली रूप से कुछ JSX को एक वेरिएबल में सेव कर सकते है और फिर कर्ली ब्रेसिज़ का उपयोग करके इसे अन्य JSX के अंदर शामिल कर सकते हैं।
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:444]: `कंडिशनली` => `कंडिशनल`

Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:444]: `के अंदर` => `में`

File: beta/src/pages/learn/conditional-rendering.md [hunk 9]
```diff
@@ -436,16 +435,16 @@ export default function PackingList() {
...
+* JSX में, `{cond ? <A /> : <B />}` का अर्थ है *"यदि `cond` है, तो `<A />` रेंडर करें, अन्यथा `<B />`"*।
+* JSX में, `{cond && <A />}` का अर्थ है *"यदि `cond`, `<A />` रेंडर करें, अन्यथा कुछ भी नहीं"*।
+* शॉर्टकट सामान्य हैं, लेकिन यदि आप सादा `if` पसंद करते हैं तो आपको उनका उपयोग करने की आवश्यकता नहीं है।
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:447]: `सादा` => `प्लेन`

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
Great work again.

Added review in the file. 

Lets close these, then you can continue with further translations

File: beta/src/pages/learn/conditional-rendering.md [hunk 10]
```diff
@@ -309,46 +309,45 @@ export default function PackingList() {
...
-
+एक [JavaScript && एक्सप्रेशन](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_AND) इसके दाईं ओर का वैल्यू देता है (हमारे केस में, चेकमार्क) यदि बाईं ओर (हमारी कंडीशन) `true` है। लेकिन अगर कंडीशन `false` है, तो पूरी एक्सप्रेशन `false` हो जाती है। React JSX ट्री में `true` को "होल" के रूप में मानता है, ठीक `null` या `undefined` की तरह, और इसके स्थान पर कुछ भी रेंडर नहीं करता है।
 
 <Gotcha>
 
-**Don't put numbers on the left side of `&&`.**
+**`&&` के बाईं ओर नंबर न लगाएं।**
 
-To test the condition, JavaScript converts the left side to a boolean automatically. However, if the left side is `0`, then the whole expression gets that value (`0`), and React will happily render `0` rather than nothing.
+कंडीशन का परीक्षण करने के लिए, JavaScript स्वचालित रूप से बाईं ओर को बूलियन में परिवर्तित कर देता है। हालाँकि, यदि बाईं ओर `0` है, तो संपूर्ण एक्सप्रेशन को वह वैल्यू (`0`) प्राप्त होता है, और React खुशी से `0` को रेंडर करेगी, न कि कुछ भी नहीं।
```
[the remark this answers is not in the document]
Comment (sanchit36) [reply beta/src/pages/learn/conditional-rendering.md:318]: `कर देगा` is better.

File: beta/src/pages/learn/conditional-rendering.md (hunk 2 above)
Comment (sanchit36) [reply beta/src/pages/learn/conditional-rendering.md:328]: `देके` sounds less complex.

**Comment by arshadkazmi42** [comment]
@sanchit36 I have verified the fixes. All looks good. You can continue further with translating the rest of the file

**Comment by sanchit36** [comment]
@arshadkazmi42  I have completed the page, need your review.

File: beta/src/pages/learn/conditional-rendering.md [hunk 11]
```diff
@@ -436,26 +435,26 @@ export default function PackingList() {
...
 
 
 
 <Challenges>
 
-### Show an icon for incomplete items with `? :` {/*show-an-icon-for-incomplete-items-with--*/}
+### अधूरे आइटम के लिए `? :` के साथ एक आइकन दिखाएं {/*show-an-icon-for-incomplete-items-with--*/}
 
-Use the conditional operator (`cond ? a : b`) to render a ❌ if `isPacked` isn’t `true`.
+एक ❌ रेंडर करने के लिए कंडीशनल ऑपरेटर (`cond ? a : b`) का इस्तेमाल करें अगर `isPacked` `true` नहीं है।
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:457]:
```
एक ❌ रेंडर करने के लिए कंडीशनल ऑपरेटर (`cond ? a : b`) का इस्तेमाल करें अगर `isPacked` `true` नहीं है।
```
=>
```
अगर `isPacked` `true` नहीं है तो एक ❌ रेंडर करने के लिए कंडीशनल ऑपरेटर (`cond ? a : b`) का इस्तेमाल करें।
```

File: beta/src/pages/learn/conditional-rendering.md [hunk 12]
```diff
@@ -533,15 +532,15 @@ export default function PackingList() {
 
 </Solution>
 
-### Show the item importance with `&&` {/*show-the-item-importance-with-*/}
+### आइटम का इम्पोर्टेंस `&&` के साथ दिखाएं {/*show-the-item-importance-with-*/}
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:535]: `का इम्पोर्टेंस` => `की अहमियत`

File: beta/src/pages/learn/conditional-rendering.md [hunk 13]
```diff
@@ -623,15 +622,15 @@ export default function PackingList() {
 
 </Sandpack>
 
-Note that you must write `importance > 0 && ...` rather than `importance && ...` so that if the `importance` is `0`, `0` isn't rendered as the result!
+ध्यान दें कि आपको `importance && ...` के बजाय `importance > 0 && ...` लिखना होगा ताकि अगर `importance` `0` हो, तो `0` परिणाम के रूप में रेंडर नहीं हो!
 
-In this solution, two separate conditions are used to insert a space between then name and the importance label. Alternatively, you could use a fragment with a leading space: `importance > 0 && <> <i>...</i></>` or add a space immediately inside the `<i>`:  `importance > 0 && <i> ...</i>`.
+इस समाधान में, नाम और इम्पोर्टेंस लेबल के बीच एक स्पेस डालने के लिए दो अलग-अलग कंडीशंस का उपयोग किया गया है। वैकल्पिक रूप से, आप एक प्रमुख स्पेस के साथ एक फ्रेगमेंट का उपयोग कर सकते हैं: `importance > 0 && <> <i>...</i></>` या `<i>` के अंदर तुरंत एक स्पेस जोड़ें: `importance > 0 && <i> ...</i>`।
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:627]: `इम्पोर्टेंस` => `importance`

Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:627]: `उपयोग कर` => `उपयोग भी कर`

Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:627]: `तुरंत एक स्पेस जोड़ें` => `शुरुआत में ही एक स्पेस जोड़ें`

File: beta/src/pages/learn/conditional-rendering.md [hunk 14]
```diff
@@ -711,9 +710,9 @@ export default function DrinkList() {
 
 </Sandpack>
 
-Here the information about each drink is grouped together instead of being spread across multiple conditions. This makes it easier to add more drinks in the future.
+यहां प्रत्येक ड्रिंक के बारे में जानकारी को कई स्थितियों में फैलाने के बजाय एक साथ समूहीकृत किया जाता है। इससे भविष्य में और ड्रिंक जोड़ना आसान हो जाता है।
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:713]: `स्थितियों` => `कंडीशंस`

Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:713]: `समूहीकृत` => `ग्रुप`

Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:713]: `जोड़ना` => `ऐड करना`

File: beta/src/pages/learn/conditional-rendering.md [hunk 15]
```diff
@@ -711,9 +710,9 @@ export default function DrinkList() {
...
 
-Another solution would be to remove the condition altogether by moving the information into objects:
+एक अन्य उपाय यह होगा कि जानकारी को ऑब्जेक्ट्स में स्थानांतरित करके कंडीशन को पूरी तरह से हटा दिया जाए:
```
Comment (arshadkazmi42) [inline beta/src/pages/learn/conditional-rendering.md:715]: `स्थानांतरित` => `मूव`

**Review by arshadkazmi42 — CHANGES_REQUESTED** [review]
I have added review for rest of the changes.

Once these are fixed. these should be go to be merged

**Review by arshadkazmi42 — APPROVED** [review]
Great work. 

Lets get this merged :+1:

**Pull request merged**